tools/heir-opt.cpp

```diff
@@ -366,18 +366,15 @@ int main(int argc, char **argv) {
       "BGV.",
       mlirToRLWEPipelineBuilder(mlir::heir::RLWEScheme::bgvScheme));
 
-  PassPipelineRegistration<mlir::heir::MlirToRLWEPipelineOptions>(
-      "mlir-to-openfhe-bgv",
-      "Convert a func using standard MLIR dialects to FHE using BGV and "
-      "export "
-      "to OpenFHE C++ code.",
-      mlirToOpenFheRLWEPipelineBuilder(mlir::heir::RLWEScheme::bgvScheme));
+  PassPipelineRegistration<mlir::heir::EntryFunctionOptions>(
+      "to-openfhe",
```

Comment:
```suggestion
      "scheme-to-openfhe",
```

I'd prefer a more specific name. At least, using `to-openfhe` makes it harder to find it via grep due to being a subset of other strings (`*-to-openfhe`).